HISTORY.rst

```diff
@@ -2,6 +2,18 @@
 History
 =======
 
+Latest
+------
+
+Major changes:
+~~~~~~~~~~~~~
+- add get_timestep and config_from_yaml functions
+
+Major changes:
```

Comment:
Probably you meant to write "Minor changes" here.

Reply:
Yup thanks.